### DISCUSSION TITLE
Rustdesk apk version for android 7

### DISCUSSION THREAD

**Discussion by arifmuradli** [rustdesk/rustdesk#5360] [Q&A]
Hello World!
I have Sunmi T2 Mini Device (pos device) that runs on Android 7.
When I searched for rustdesk apk, I found only [1.1.10-1] version. I was absolutely terrible and blurry. However as I see latets version is 1.2.1 and it works fine on my smartphone.
The Sunmi Device is not too old and anydesk there works just fine with ideal visual quality (of course, with its unstableness).
Maybe you know with version is the latest compatible with android 7?

**Comment by xlionjuan** [comment]
It is non sense to support 7+ years of operating system.
